src/schema/types/mutations/Portfolio.ts

```diff
@@ -1,15 +1,31 @@
 import { arg, nonNull, intArg } from "nexus";
 
+import { context } from "../../../context";
```

Comment:
import 해올때는 상대 경로보다 절대 경로를 사용하면 더 보기 편할것 같아요
```suggestion
import { context } from "context";
```